src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,35 @@
 import React from 'react';
+import './PostInfo.scss';
+import { Posts } from '../../types/Posts';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+type Props = {
+  post: Posts,
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
```

Comment:
You did a good job by destructing props for comment at `CommentInfo` component, could you do it also here?